modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -1,52 +1,56 @@
 <template>
-  <div class="apos-login apos-theme-dark">
-    <div class="apos-login__overlay"></div>
-    <div class="apos-login__menu-overlay"></div>
-    <AposLoginBackground />
+  <transition name="fade-stage">
+    <div class="apos-login apos-theme-ssdark" v-show="loaded">
```

Comment:
`apos-theme-ssdark` is it a typo? or do you want just to disable the dark theme?

Reply:
oh yes, was testing light/default